handbook/support/p4-enablement.md

```diff
@@ -0,0 +1,165 @@
+
+Specifying views in your client configuration allows you to declare which files from the perforce depot are relevant to you, or which files you want to be part of your *workspace*. You can learn more about this topic [here](https://www.perforce.com/perforce/doc.973/cmdguide/html/details.htm).
+
+Once you've created a depot on dogfood, map a local directory to it by adding an entry to your client spec under `Views:`
+
+`//<depot name>/... //<client name>/<root directory>/...`
+
+Note that the client name is mapped to your `Root:` setting in the client spec.
+
+### Add files and submit
+
+Once you've created a depot on the server, and created a client spec, adding files is a lot like git. To add the files run `p4 add <relative path to files>/...` for example while in `/Users/warrengifford` I could run `p4 add ./LifeBox/...`. This will open a change spec for which you must provide a description:
+
+```
+Change: new
+
+Client: warren
+
+User:   admin
+
+Status: new
+
+Description:
+        <enter description here>
+
+Files:
+        //support-tools/.git/COMMIT_EDITMSG     # add
+        //support-tools/.git/FETCH_HEAD # add
+        //support-tools/.git/HEAD       # add
+        //support-tools/.git/ORIG_HEAD  # add
+        //support-tools/.git/config     # add
+        //support-tools/.git/description        # add
+
+        # etc etc
+```
+You can see the files staged for adding with `p4 opened`.
+
+Finally run `p4 submit` to load the files to the depo on the server.
+
+### Configuring Sourcegraph to sync dogfood depots
+
+Our [documentation](https://docs.sourcegraph.com/admin/repo/perforce) covers the requirements to sync to Sourcegraph, however for convienience it should be noted we have a user called `buildkite` on our dogfood instance who session ticket will not expire. To generate the ticket for this account reference our shared [1Password](https://team-sourcegraph.1password.com/vaults/dnrhbauihkhjs5ag6vszsme45a/allitems/lajspc6a5valfbsh3whpcb5bp4).
+
+To learn more about general p4 commands checkout this resource:
+[https://www.perforce.com/perforce/doc.973/cmdguide/html/quicksta.htm](https://www.perforce.com/perforce/doc.973/cmdguide/html/quicksta.htm)
+
+# Local Perforce server
+
+Joe has also developed an awesome perforce server [image](https://github.com/sourcegraph/helix-docker/blob/main/helix-p4d/Dockerfile) that allows for local deployment of a perforce server.
+ 
+The following procedure runs the image:
+
+1. Start the server `docker run --publish 1666:1666 sourcegraph/helix-p4d:2020.2`
+
+2. Open a new terminal, configure the terminal session env:
+```
+export P4PORT=:1666
+export P4USER=admin
+```
+3. ping the server from a separate terminal with p4 ping and enter the password below:
+```
+ARG P4PASSWD=pass12349ers
```

Comment:
```suggestion
pass12349ers
```

I am concerned it would confuse people that "ARG P4PASSWD=" is also part of the password.